src/components/About/About.tsx

```diff
@@ -1,159 +1,153 @@
-            <Row style={paragraphStyle}>{t("about:visionContent")}</Row>
-            <Row style={{ width: "100%", textAlign: "center" }}>
-                <Typography.Title style={{ width: "100%" }} level={3}>
-                    {t("about:missionTitle")}
-                </Typography.Title>
-            </Row>
-            <Row style={paragraphStyle}>{t("about:missionContent")}</Row>
-            <Row style={{ width: "100%", textAlign: "center" }}>
-                <Typography.Title style={{ width: "100%" }} level={3}>
-                    {t("about:valuesTitle")}
-                </Typography.Title>
-            </Row>
-            <Row style={paragraphStyle}>
+                            https://github.com/AletheiaFact/aletheia
+                        </a>
+                    </>
+                }
+                action={
+                    <Button
+                        variant="contained"
+                        size="small"
+                        startIcon={<DescriptionIcon />}
+                        href="https://docs.google.com/viewer?url=https://raw.githubusercontent.com/AletheiaFact/miscellaneous/290b19847f0da521963f74e7947d7863bf5d5624/documents/org_legal_register.pdf"
+                        target="_blank"
+                        rel="noreferrer"
+                        sx={{
+                            position: "absolute",
+                            bottom: "15px",
+                            right: "15px",
+                        }}
+                    >
+                        {t("about:labelButton")}
+                    </Button>
+                }
+            />
+            <Box sx={{ textAlign: "center", width: "100%" }}>
+                <Typography variant="h4">{t("about:title")}</Typography>
+            </Box>
+            <Box sx={paragraphStyle}>
+                <Typography>{t("about:intro")}</Typography>
+            </Box>
+            <Box sx={{ textAlign: "center", width: "100%" }}>
+                <Typography variant="h5">{t("about:visionTitle")}</Typography>
+            </Box>
+            <Box sx={paragraphStyle}>
+                <Typography>{t("about:visionContent")}</Typography>
+            </Box>
+            <Box sx={{ textAlign: "center", width: "100%" }}>
+                <Typography variant="h5">{t("about:missionTitle")}</Typography>
+            </Box>
+            <Box sx={paragraphStyle}>
+                <Typography>{t("about:missionContent")}</Typography>
+            </Box>
+            <Box sx={{ textAlign: "center", width: "100%" }}>
+                <Typography variant="h5">{t("about:valuesTitle")}</Typography>
+            </Box>
+            <Box sx={paragraphStyle}>
                 <ul>
-                    <li key={1}>{t("about:valueAccessibility")}</li>
-                    <li key={2}>{t("about:valueCredibility")}</li>
-                    <li key={3}>{t("about:valueAutonomy")}</li>
+                    <li>{t("about:valueAccessibility")}</li>
```

Comment:
question: wouldn't removing the keys create a warning and render conflict in react?